templates/bundles/PrestaShopBundle/Admin/Sell/Order/Order/Blocks/Create/customer.html.twig

```diff
@@ -163,6 +163,7 @@
       <div class="card-body">
         <p class="mb-0 js-customer-email"></p>
         <p class="mb-0 js-customer-birthday"></p>
+        <p class="mb-0 js-customer-company"></p>
```

Comment:
Shouldn't be displayed only when b2B is enabled?

Reply:
Done!